### ISSUE TITLE
date in publicationStmt

### ISSUE THREAD

**Issue by eu-genia** [BetaMasaheft/Documentation#2085]
Old records created from the index have a `date `element within `publicationStmt `for the date of initial transformation
![image](https://user-images.githubusercontent.com/17987728/164695007-e681387b-d333-4b09-b3f8-2482bd742753.png)

Newer records do  not have this element as it has been replaced by the earliest `change `element. I suggest that when we edit older files, when we remember to do so, we remove this unnecessary `date `element (one could eventually do the batch edit but probably this is not important enough for the effort)
@thea-m @DenisNosnitsin1970

**Comment by thea-m** [comment]
I agree it would be better to delete it, but it is very impractical to keep many of such small to-do items in mind whenever opening some old record. 
Would it be safe to delete all `<date>`s in `<publicationStmt>` beginning with 2016? This can be done with find/replace (in manageable batches, of course).

**Comment by eu-genia** [comment]
(if you forget to remove it it is fully OK, it is not an error, just some noise - but at least not copy paste it in new records would already be good)

**Comment by eu-genia** [comment]
I will refactor the date and license once we have a new release
